qt/widgets/common/CMakeLists.txt

```diff
@@ -904,11 +912,20 @@ mtd_add_qt_tests (TARGET_NAME MantidQtWidgetsCommonTest
 # This file is NOT packaged. See installers/WinInstaller for the package
 # version.
 ###########################################################################
-if ( WIN32 )
+if ( WIN32 AND ENABLE_MANTIDPLOT)
   set ( QT_CONF qt.conf )
   configure_file ( qt.conf.template ${CMAKE_CURRENT_BINARY_DIR}/${QT_CONF} @ONLY )
   add_custom_command ( TARGET MantidQtWidgetsCommonQt4 POST_BUILD
                        COMMAND ${CMAKE_COMMAND} ARGS -E copy_if_different
                                ${CMAKE_CURRENT_BINARY_DIR}/${QT_CONF}
                                ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${CMAKE_CFG_INTDIR}/${QT_CONF} )
 endif()
+
+if ( WIN32 AND ENABLE_WORKBENCH)
```

Comment:
Does this work/Is it necessary? We didn't use `qt.conf` before now as Qt only picks it up from the application executable directory and for us this is now Python and not our application.

Reply:
Sorry, don't quite know (works, necessary, etc.).

Should I just leave it as `if (WIN32)` and remove the separate flags for workbench and mantidplot? Just not sure and don't have a WIN32 system to check this on.